cmd/podman/containers/ps.go

```diff
@@ -56,10 +57,10 @@ func init() {
 func listFlagSet(flags *pflag.FlagSet) {
 	flags.BoolVarP(&listOpts.All, "all", "a", false, "Show all the containers, default is only running containers")
 	flags.StringSliceVarP(&filters, "filter", "f", []string{}, "Filter output based on conditions given")
+	flags.BoolVar(&listOpts.External, "external", true, "Show containers in storage not controlled by Podman")
```

Comment:
Think we should use `--storage` here for consistency with `podman rm --storage`

Reply:
I am fine with this.